epub33/core/index.html

```diff
@@ -4092,348 +4092,376 @@ <h4>Prefixed Properties</h4>
 					<section id="sec-css-prefixed-writing-modes">
 						<h5>CSS Writing Modes</h5>
 
```

Comment:
The intro paragraph of §3.3.3 is clear for a reader who is already familiar with EPUB, more exactly with earlier versions of EPUB 3. For a newcomer, who gets into the world of EPUB via EPUB 3.3 this paragraph, and the whole section is really cryptic and comes out of the blue...

I think some explanatory words on how these properties came about, that this specification includes this section to ensure backward compatibility, etc, would be necessary. It should also suggest (there are already references to this) that new authors for new content should probably ignore the whole section altogether. Etc.

Reply:
This is unchanged from EPUB 3.2. I will add an introductory sentence or two, but is the existing text good enough for suggesting these shouldn't be used today? 

> EPUB Creators are advised to use CSS-native solutions for the removed properties where and when they are available.

> EPUB Creators currently using these prefixed properties are advised to move to unprefixed versions as soon as support allows...

Reply:
SHOULD NOT for prefixed properties in general? That would trigger an EPUBCheck warning, which would break backward compatibility. 

I also think these are relatively easy to continue supporting--some are even built into browsers.

Reply:
> That would trigger an EPUBCheck warning, which would break backward compatibility.

That is what I was worried about. We should not change it then.